views/AdminView/components/BotNameInput.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC, useCallback, useState } from 'react';
+import React from 'react-native';
+import {
+  Button,
+  Input,
+  Typography,
+  useInputValue,
+} from '../../../.storybook/stories';
+import { useAppSelector } from '../../../redux/hooks';
+import { sendIoTMessage } from '../../../utils';
+
+type Props = {
+  botFatherAccessHash: string;
+  botFatherId: string;
+};
+
+export const BotNameInput: FC<Props> = ({
+  botFatherAccessHash,
+  botFatherId,
+}) => {
+  const [isSubmitting, setIsSubmitting] = useState(false);
+
+  const [botUsername, onBotUsernameChange] = useInputValue();
+
+  const { isWaitingForUsername, isUsernameInvalidError, isUsernameTakenError } =
+    useAppSelector(state => state.admin);
+
+  const onConfirmBotUsername = useCallback(async () => {
+    setIsSubmitting(true);
+    await sendIoTMessage(botUsername, botFatherAccessHash, botFatherId);
+    setIsSubmitting(false);
+  }, [botUsername, botFatherAccessHash, botFatherId]);
+
+  return (
+    <>
+      <Typography
+        variant="body-medium"
+        text="Podaj username dla bota, którego będziesz uzywał na Raspberry PI. Jego nazwa musi kończyć się na bot, np. HomeBot lub home_bot"
```

Comment:
I would add e.g. `_bot` at the end without requesting it from user

Reply:
We can do this in the future, but for now, the user has to provide the bot name in the `.env` file, so I don't want to make it inconsistent.